_health-care/_js/components/HealthCareApp.jsx

```diff
@@ -214,8 +214,30 @@ class HealthCareApp extends React.Component {
     this.setState({ applicationData: newApplicationData });
   }
 
+  resetNullValues(objectData) {
+    return _.mapValues(objectData, (value, _key) => {
```

Comment:
Having another problem with `_.mapValues`: because it returns everything as an object, when it gets to `children` or `providers`, which should be an array, it turns it into an object. This means that later functionality in `GrowableTable` where it maps through the array of children or providers fails. I've tried to figure out a way for `_.mapValues` to check if the object is an array, and if so to map through the array and called itself recursively on each element in the array so as to not touch the array, but it is still returning an object for `children` and `providers`. Not sure another way around this.